packages/components/src/FilterBar/FilterBar.scss

```diff
@@ -83,6 +83,11 @@ $tc-filter-bar-width: 250px !default;
 	background-color: $tc-filter-bar-highlight-color;
 }
 
+:global(.tc-list-toolbar) .filter {
```

Comment:
Shouldn't this be defined in list toolbar as it's a component placement within toolbar ?

Reply:
AFAIK @frassinier request on that point to use this approach to list all context where a component is used in the component.
So when you update a component you see all the context where is used and modified

Reply:
@jmfrancois I don't remember but if it's what I said, I would like to correct by the opposite: I prefer not using `:global`statement at all, and move it to ListToolbar.
With that, if you remove List (and its Toolbar), you can forget this piece of code and embed dead code.